src/test/groovy/graphql/parser/ParserTest.groovy

```diff
@@ -69,11 +71,11 @@ class ParserTest extends Specification {
 
 
```

Comment:
This class was littered with IDEA warnings.  I clean up a lot because broken windows and all that